app/metrics.yaml

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+no_lint:
+  - CATEGORY_GENERIC
+
+metrics:
+  default_browser:
+    type: boolean
+    lifetime: application
+    description: |
+      Is Focus the default browser?
+    send_in_pings:
+      - metrics
```

Comment:
If you're only planning on sending this in the `metrics` ping, you can safely remove this property ,since it goes there by default.

Reply:
Would this still be true if I follow the above suggestion to rename this grouping from `metrics` to `browser`?

Reply:
Yes, that would still be true ([see our docs](https://mozilla.github.io/glean/book/reference/yaml/index.html#optional-metric-parameters)): the `metrics` entry in the `send_in_pings` property refers to the Glean-owned [metrics ping](https://mozilla.github.io/glean/book/user/pings/metrics.html), while the `metrics` category used to group the metrics (the own our linter complains about) is just a label. We know that this might be confusing and that's why are linter complains about it :-P

It's been the source of confusion in Fenix too, so changing it here would be for the best!